doc/dev/index.rst

```diff
@@ -179,6 +179,100 @@ with::
 
 For more ``vstart.sh`` examples, see :doc:`/dev/quick_guide`.
 
+What is merged where and when ?
+===============================
+
+Commits are merged into branches according to criterions that change
+during the lifecycle of a Ceph release. This chapter is the inventory
+of what can be merged in which branch at a given point in time.
+
+Development releases (i.e. x.0.z)
+---------------------------------
+
+What ?
+^^^^^^
+
+ * features and bug fixes
+
+Where ?
+^^^^^^^
+
+Features are merged to the master branch. Bug fixes should be (but it is not mandatory)
+merged in the branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.).
+
+When ?
+^^^^^^
+
+After the stable release candidates of the previous release starts phase 2 (see below).
+
+
+Stable release candidates (i.e. x.1.z) phase 1
+----------------------------------------------
+
+What ?
+^^^^^^
+
+ * bug fixes only
+
+Where ?
+^^^^^^^
+
+The branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.) or master.  All commits in master will be merged in the
```

Comment:
Bug fixes should be merged to the named branch corresponding to the stable release candidate (e.g. "jewel" for `10.1.z`) or to master.